projects/Flux/onnx/polyaxonfile.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Same thing here, I think we can just use the same notebook yaml in the `polyaxon` folder since all of the code is in the same repo

Reply:
This one actually had to be different because of the branch :)

Reply:
It's best practice to not commit a yaml with a branch since this will be merged into master and then it will be there. The branch should be a manual thing the user does when working with the yaml

Reply:
:+1: